### ISSUE TITLE
On January 31, 2019 at 4:55 pm Eastern time, google.load('visualization', '1', {packages: ['linechart']});, blocked my javascript WebApp

### ISSUE THREAD

**Issue by ksben** [google/google-visualization-issues#2715]
On January 31, 2019 at approx. 4:55pm eastern time, a coworker discovered that my webApp failed to completely load.  In the process of debugging, I discovered that the WebApp's javascript code could not make it past a line that contained the following:

google.load('visualization', '1', {packages: ['linechart']});

Once I removed this line, the webApp continued to load but stopped again at another point beyond this line.  This to me indicated that there was something wrong with that service.  At approx 5:45 pm that same day, I discovered that the webApp was no longer being blocked and has been running fine since (as far as I know).

I have two questions: 
1) Was there an issue with the google.load service on January 31, 2019 at Approx 4:55 pm?
2) What is the best strategy in capturing/handling a google.load service issue?

**Comment by dlaliberte** [comment]
Yes, there was an "issue" with the google.load service, which was actually an intentional "drain test" of the deprecated service where you load the jsapi loader from.  Your page actually failed to load jsapi as a result.  

There was a belated (mid-stream) announcement of this outage posted [here](https://groups.google.com/forum/#!topic/google-visualization-api/hGys_iI99bE).

You'll note the larger issue is that the service that loads the jsapi loader is going to be turned off entirely, but that we will deploy a redirect to the new loader before that happens.  You will be effectively updated to v46.  

There will probably be another drain test before the transition is complete, and regardless, the upgrade may be problematic for your particular charts, so you should update your charts to use the new loader as soon as possible, as documented [here](https://developers.google.com/chart/interactive/docs/basic_load_libs#update-library-loader-code).

**Comment by dlaliberte** [comment]
If you want to load your page in such a way that a failure to run google.load() or google.charts.load() does not block the rest of your page, then you'll need to run google.load or google.charts.load in a separate JavaScript thread, which you can do with a window.setTimeout, or perhaps in a try-catch block.
